share/man/man9/pmap_kextract.9

```diff
@@ -0,0 +1,58 @@
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.\" Copyright (c) 2023 The FreeBSD Foundation
+.\"
+.\" This manual page was written by Mina Galić <FreeBSD@igalic.co> under
+.\" sponsorship from the FreeBSD Foundation.
+.\"
+.Dd August 24, 2023
+.Dt PMAP_KEXTRACT 9
+.Os
+.Sh NAME
+.Nm pmap_kextract ,
+.Nm vtophys
+.Nd extract from kernel page table to a physical address
+.Sh SYNOPSIS
+.In sys/param.h
+.In vm/vm.h
+.In vm/pmap.h
+.Ft vm_paddr_t
+.Fo pmap_kextract
+.Fa "vm_offset_t va"
+.Fc
+.Ft vm_paddr_t
+.Fo vtophys
+.Fa "vm_offset_t va"
+.Fc
+.Sh DESCRIPTION
+The
+.Fn pmap_kextract
+function, and its alias
+.Fn vtophys ,
+map a kernel virtual address to a physical page.
+.Sh RETURN VALUES
+The
+.Fn pmap_kextract
+function will return the physical page address
+.Ft ( vm_paddr_t )
+associated with the kernel virtual address
+.Fa va
+inside the physical map of the address space.
+In case of failure,
```

Comment:
Please take a look at how this is done in pmap_extract.9 that you are referencing -- there should be a `.Sh RETURN VALUES` section documenting what is returned on success and failure (e.g. address and NULL).  Also I don't think saying it will panic (or fail assertion) makes sense, it shouldn't do that and looking at the code it does not for bad value issues (something being unimplemented or missing from hardware is out of scope of this man page).

Reply:
I agree here, it would be enough to say that zero/NULL is returned in the case of failure.

Better yet, we should state explicitly in the description the requirement that `va` **must** represent a "valid" kernel address. As in, the value of `va` is obtained by legitimate means (e.g. a pointer value from a malloc'd object). To pass an arbitrary/random value within the KVA range _might_ appear to succeed (if the virtual mapping exists), but it is a programming error subject to "undefined behavior" such as a kernel panic.

I've included this in my suggested text above.

Reply:
Thank you for your review.
I've incorporated your changes, and updated this section, too.
If you're still unhappy with it, we can scrap the entire In case of failure paragraph.